examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    fig, ax=plt.subplots(1,3,figsize=(20,20))
</pre><p>20x20 inches is insane for a figure I think, this will create a huge figure</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Consider that they are pretty high-resolution images, so when you display them as (20,20), they look quite nice.